navigation/localization/auv_localization/auv_localization/model_ekf.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import rclpy
+from sensor_msgs.msg import NavSatFix, Imu
+from rclpy.node import Node
+
+import random
+
+class EKFModel_ImageFeedback(Node):
+
+    def __init__(self, i):
+        super().__init__(i)
+
+        self.drone_acc = np.zeros([3,])
+        self.timestep = 0
+        self.M = np.zeros([1,6])
+        self.M[0,2] = 1
+        self.C = None
+        self.A = np.zeros(6)
+        self.B = np.zeros([6,3])
+
+        # Camera intrinsic parameters(values taken from unity editor)
+        self.f_x = 20.78461*640/36  # Focal length in pixels (x-direction)
```

Comment:
can these be expected from the camera_info topic instead? 
camera_info message has these fields. so read these from that (add that topic to topics msg too) and modify the drone prefab's camera_info publisher with these values too. then the source of truth is the sim or the real vehicle.

Reply:
Ill do this, seems more structured and less hardcoded